qucs/qucs/dialogs/matchdialog.cpp

```diff
@@ -238,40 +252,51 @@ void MatchDialog::slotSetTwoPort(bool on)
   if(on) { // two-port matching ?
     S11Label->setText(tr("S11"));
     S21Label->setText(tr("S21"));
-    S12magEdit->setEnabled(true);
-    S22magEdit->setEnabled(true);
-    S12degEdit->setEnabled(true);
-    S22degEdit->setEnabled(true);
-    S12Label->setEnabled(true);
-    S22Label->setEnabled(true);
-    S12sLabel->setEnabled(true);
-    S22sLabel->setEnabled(true);
-    S12degEdit->setEnabled(true);
-    S22degEdit->setEnabled(true);
-    S12uLabel->setEnabled(true);
-    S22uLabel->setEnabled(true);
-    Port2Label->setEnabled(true);
-    Ref2Edit->setEnabled(true);
-    Ohm2Label->setEnabled(true);
+    S12magEdit->setVisible(true);
```

Comment:
can you put these in a seperate function?

    void something(bool x)
    {
      S12magEdit->setVisible(x);
      // the other 13 lines
    }

better yet: use a container

    for(x in list){ x->setEnabled(true); }

(just a thought)